app/src/main/java/com/ucmobiledevelopment/freeways/ReportIncidentActivity.kt

```diff
@@ -28,7 +28,7 @@ class ReportIncidentActivity : ComponentActivity() {
                 }
                 val context = LocalContext.current
                 ExtendedFloatingActionButton (
-                    text = {  Text(text = "Back") },
+                    text = { Text(text = "Back") },
```

Comment:
Incredibly small change, just removed an extra space because I happened to notice it.